org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/services/extensions/AggregetedHoverValuesTest.java

```diff
@@ -0,0 +1,93 @@
+/**
+ *  Copyright (c) 2018 Angelo ZERR.
```

Comment:
Add Liferay header or your name

Reply:
OK, @angelozerr , one more thing. 
I found if we only use one System.lineSeparator(), it dees'nt work. And if I use two, it will looks much better.
![Screenshot (24)](https://user-images.githubusercontent.com/24595644/67380187-8b39f980-f5bc-11e9-92d9-74975eb55076.png)
![Screenshot (26)](https://user-images.githubusercontent.com/24595644/67380198-8f661700-f5bc-11e9-96e1-4e9bfde5adb1.png)
![1571823642(1)](https://user-images.githubusercontent.com/24595644/67380211-912fda80-f5bc-11e9-9eed-64503464a37c.png)

Reply:
How do you think, should we use two line separator?

Reply:
It seems LSP4E required that but what about with vscode?

Reply:
Sorry, just saw it. I believe it won't hurt anything on vscode. If someday we find some problems on vscode which caused by it, we can go back here, and discuss about it.